functions/__fzf_search_git_log.fish

```diff
@@ -1,14 +1,11 @@
 # Originally implemented in and transposed from https://github.com/patrickf3139/dotfiles/pull/2
 function __fzf_search_git_log --description "Search the git log of the current git repository. Insert the selected commit hash into the commandline at the cursor."
-    if not git rev-parse --git-dir >/dev/null 2>&1
+    # see documentation for git format placeholders at https://git-scm.com/docs/git-log#Documentation/git-log.txt-emnem
+    # %h gives you the abbreviated commit hash, which is useful for saving screen space, but we will have to expand it later below
+    if not set --local gitLog (git log --color=always --format=format:'%C(bold blue)%h%C(reset) - %C(cyan)%as%C(reset) %C(yellow)%d%C(reset) %C(normal)%s%C(reset)  %C(dim normal)[%an]%C(reset)' 2>/dev/null)
```

Comment:
Could we hold off on the camelCase change? Sorry but I'm not quite sold on it now. Prefer snake_case for now since it's more consistent with other fish code.

Reply:
For sure, that was a mistake as I mentioned above.

Reply:
Ah I see...I saw it but wasn't sure if you were planning on keeping it in the PR.